paasta_tools/deployd/watchers.py

```diff
@@ -288,6 +288,8 @@ def filter_event(self, event):
         starts_with = ['marathon-', 'deployments.json']
         if any([event.name.startswith(x) for x in starts_with]):
             return event
+        elif '.json' in event.name and event.path.split('/')[-1] == 'secrets':
+            return event
```

Comment:
I think this function should return a boolean indicating whether it's interesting or not, instead of returning the event or nothing (I know that you've not changed that in this PR, but its worth fixing whilst we're here).

Reply:
sgtm